pkg/api/server.go

```diff
@@ -361,11 +361,11 @@ func redactValues(req interface{}, keys, sensitiveSubstrings []string) interface
 
 func loggingUnaryInterceptor(logger *zerolog.Logger) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
Add a comment here about needing deep copy for true redaction and then feel free to merge